src/region/RegionAnnotations.tsx

```diff
@@ -29,15 +29,40 @@ type State = {
     rectRef?: RegionRectRef;
 };
 
+const handleEvent = (event: Event): void => {
+    event.preventDefault();
+    event.stopPropagation();
+};
+
 export default class RegionAnnotations extends React.PureComponent<Props, State> {
     static defaultProps = {
         annotations: [],
         isCreating: false,
         isRotated: false,
     };
 
+    popupReplyRef: React.RefObject<HTMLDivElement> = React.createRef();
+
     state: State = {};
 
+    componentWillMount(): void {
+        if (!this.popupReplyRef.current) {
+            return;
+        }
+
+        this.popupReplyRef.current.removeEventListener('mousedown', handleEvent);
+        this.popupReplyRef.current.removeEventListener('mouseup', handleEvent);
+    }
+
+    componentDidUpdate(): void {
+        if (!this.popupReplyRef.current) {
+            return;
+        }
+
+        this.popupReplyRef.current.addEventListener('mousedown', handleEvent);
+        this.popupReplyRef.current.addEventListener('mouseup', handleEvent);
```

Comment:
Why we add a listener for every update? But we only remove once when unmount. Will this causes multiple listeners?

Reply:
Good catch, I'll add some logic around that